docs/events/api/timeline-api-get.asciidoc

```diff
@@ -0,0 +1,86 @@
+[[timeline-api-get]]
+=== Get Timelines or Timeline templates
+
+Get Timelines or Timeline templates.
+
+==== Request URL
+
+`GET <kibana host>:<port>/api/timelines`
+
+==== Request params
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`only_user_favorite` |Boolean a|Returns only favorite Timelines / Timeline templates if set to `true`. Default to `false`
+|No
+|`page_index` |Number |Page number of the result
+|No
+|`page_size` |Number |Items per page in result
+|No
+|`search` |String |The key word of the Timelines / Timeline templates you are searching for
+|No
+|`sort_field` |String |The order you want the result to be arranged. Can be `title`, `description`, `updated`, `created`
+|No
+|`sort_order` |String |`asc` or `desc`
+|No
+|`status` |String |Can be `active` or `immutable`. `active` returns all the custom Timelines / Timeline templates, whereas `immutable` returns Elastic prebuilt templates.
```

Comment:
Recommend removing the period at the end of this line since none of the other table entries in this column have periods.